fhir-bulkimportexport-webapp/pom.xml

```diff
@@ -116,7 +116,7 @@
                             <goal>jar</goal>
                         </goals>
                         <configuration>
-                            <classifier>jar</classifier>
+                            <classifier>client</classifier>
```

Comment:
what is this for?

Reply:
The classifier enables a jar to be generated so I can pickup the jar and put it in bulkdata-operation. 
turns out the jar on the left just appends -jar.jar in the example case on maven-jar-plugin they use classifier client which generates -client.jar